core/livepeernode.go

```diff
@@ -38,7 +38,7 @@ var MaxSessions = 10
 type NodeType int
 
 const (
-	BroadcasterNode NodeType = iota
+	BroadcasterNode NodeType = iota + 1
```

Comment:
Can we define an actual enum at the end of this list instead of changing the initial numbering here?

Reply:
I had initially defined something like `ServiceNode` , but in the end it felt cleaner to stick with the current existing enums and have a standalone redeemer deployment not have " node type" for now.

Reply:
Why not a "RedeemerNode" ?

Reply:
in the future it could also be a Redeemer + Blockwatcher + RoundInitialiser , so to not get caught up on naming I decided to just make a zero value

Reply:
There is existing code that maps a node type to string - here is an [example](https://github.com/livepeer/go-livepeer/blob/ed7f24c763ad8db4592db5b4d1a2f2ce83aa6ca0/cmd/livepeer/livepeer.go#L280). So, a new node type should have an associated name + string representation. Otherwise, when a redeemer is run, based on the code in the linked example, the node will call `InitCensus()` with nodeType = "bctr".

If the worry is around changing the node type name in the future (FWIW this is just an internally used value so I don't think its a big deal to use a find + replace to update the name in the future if needed), then we could just use the following check that doesn't rely on a node type:

```golang
if !*redeemer {
    // Do something if the -redeemer=false
}
```

Reply:
> Redeemer + Blockwatcher + RoundInitialiser

That seems okay - a "RedeemerNode" could also implicitly run the blockwatcher + round initializer. If we ever needed to make one of these standalone by default, we can also add those as separate node types, eg "BlockwatcherNode" . But that doesn't preclude a RedeemerNode also being a blockwatcher.

We do similar for OT mode, where a `-transcoder` alone is a standalone TranscoderNode, while an `-orchestrator -transcoder` is an OrchestratorNode that implicitly runs a transcoder.

Reply:
Ok let's go with an additional definition then , that seems like the best way forward from here.

Reply:
fixed in d5993ce